src/FSharpPlus/Data/NonEmptySeq.fs

```diff
@@ -463,7 +463,7 @@ module NonEmptySeq =
     ///
     /// <returns>The result sequence.</returns>
     let zip3 (source1: NonEmptySeq<_>) (source2: NonEmptySeq<_>) (source3: NonEmptySeq<_>) = Seq.zip3 source1 source2 source3 |> unsafeOfSeq
-
+    
```

Comment:
Just one more thing. Is there a way you can remove this change? I mean, in such a way that this file is not touched at all by this PR. Sorry for nitpicking, it's just to keep our history (which is about 10 years old) clean.

Reply:
Sorry, my IDE sometimes messes up whitespace. Fixed :-)